<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Schedule_Menu</class>
    <widget class="QWidget" name="Schedule_Menu">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>760</width>
                <height>680</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QPushButton" name="save_button">
            <property name="geometry">
                <rect>
                    <x>330</x>
                    <y>500</y>
                    <width>100</width>
                    <height>35</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Save</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="url_input">
            <property name="geometry">
                <rect>
                    <x>230</x>
                    <y>350</y>
                    <width>300</width>
                    <height>45</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="inputMask">
                <string />
            </property>
            <property name="text">
                <string />
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
                <string>https://www.example.com/</string>
            </property>
        </widget>
        <widget class="QLabel" name="url_label">
            <property name="geometry">
                <rect>
                    <x>255</x>
                    <y>300</y>
                    <width>250</width>
                    <height>45</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>URL</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="time_label">
            <property name="geometry">
                <rect>
                    <x>255</x>
                    <y>180</y>
                    <width>250</width>
                    <height>45</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Time (24h Format)</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="time_input">
            <property name="geometry">
                <rect>
                    <x>230</x>
                    <y>230</y>
                    <width>300</width>
                    <height>45</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="inputMask">
                <string />
            </property>
            <property name="text">
                <string />
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
                <string>12:00</string>
            </property>
        </widget>
        <widget class="QPushButton" name="exit_button">
            <property name="geometry">
                <rect>
                    <x>640</x>
                    <y>620</y>
                    <width>90</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Exit</string>
            </property>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>